index.html

```diff
@@ -163,16 +163,37 @@ <h2>About me</h2>
                 </div>
             </section>
 
-             <div>
+            <section class="form-section">
+             <form action="https://formspree.io/f/mjvzwlqy" method="post">
                 <h2 class="touch">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-                <br>
-                <div class="textareas">
-                <textarea name="name" id="" cols="41" rows="2">Full name</textarea>
-                <textarea name="e-mail" id="" cols="41" rows="2">E-mail address</textarea>
-                <textarea name="text" id="" cols="41" rows="10">Enter text here</textarea>
-                <div class="butt"><button class="btn" type="button"> Get in touch</button></div>
-                </div>
-            </div>
+                <ul>
+                    
+                    <li >
+                        <label for="name">
+                            <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                        </label>
+                    </li>
+                    <li>
+                        <label for="mail">
+                            <input type="e-mail" id="mail" name="user_email" placeholder="Email address" required>
+                        </label>
+
+                    </li>
+                    <li>
+                        <label for="msg">
+                            <textarea name="user_message" id="msg" cols="30" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
+                        </label>
+                    </li>
+                    <li class="button">
+                        <button type="submit" class="btn">Get in touch</button>
+                    </li>
+
+
```

Comment:
- [ ] Kindly make sure all your inputs design stick to the ones from the Figma template (e.g: **Adding the background image, adding the missing social media icons, using the correct padding and color in each input_text, all the inputs fields should be centered by using `flexbox`,...** ). Check all the details from Figma to update your design.

| Figma Design | Your Design |
| ------------------ | ---------------- |
| ![Screenshot from 2022-09-27 14-27-28](https://user-images.githubusercontent.com/83042742/192528594-a84e3313-d80b-4e7c-9c14-96990af80a9d.png) | ![Screenshot from 2022-09-27 14-24-22](https://user-images.githubusercontent.com/83042742/192528242-f5449401-0557-45cc-b386-a7fd054729a3.png) |